apps/frontend/src/renderer/components/debug/LogViewer.tsx

```
@@ -0,0 +1,125 @@
+        return 'text-foreground';
+    }
+  };
+
+  return (
+    <div className="flex flex-col h-full gap-4">
+      {/* Controls */}
+      <div className="flex items-center gap-4">
+        <div className="flex-1">
+          <label className="text-sm font-medium mb-2 block">
+            {t('logs.sourceLabel')}
+          </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)